recipes/scala/pom.xml

```diff
@@ -27,6 +27,15 @@
 			</releases>
 		</repository>
 	</repositories>
+
+	<!-- Enable this for staged release of the artifact
+	<repositories>
+		<repository>
+			<id>staged-releases</id>
+			<url>https://oss.sonatype.org/content/repositories/comadobeplatform-1037</url>
+		</repository>
+	</repositories> -->
+	
```

Comment:
"staged release" - does this refer to a traditional "Stage" version?  1037 is dynamic so should it be here?  Can we shorten the `url` to  just `https://oss.sonatype.org/content/repositories`

Reply:
comadobeplatform-1037 is the released version of the authoring-sdk_2.11. 
Added it in pom just for reference for future iterations of releases and had it commented out. The url has to refer to this whole path as per the Sonatype's location of the artifact.

Reply:
Nice to have 'help' in the form of this comment so dev can be easier going forward.